tests/timetables/test_assets_timetable.py

```diff
@@ -142,6 +142,7 @@ def test_serialization(asset_timetable: AssetOrTimeSchedule, monkeypatch: Any) -
                     "uri": "test://asset/",
                     "group": "asset",
                     "extra": {},
+                    "watchers": [],
```

Comment:
Serialized DAG space is "precious" -- if it's empty/default it should not appear in the serialized JSON please.

Reply:
That makes sense! I removed it if there is no watcher